tests/e2e/specs/mutation.cy.js

```diff
@@ -199,4 +199,19 @@ describe('Mutations component', () => {
       .get('@errTooltip')
       .should('not.have.css', 'display', 'none')
   })
+  it('should display action buttons within viewport', () => {
+    mockMutations()
+    openMutationsForm('checkpoint')
+    cy.get('.v-card-actions').eq(0).then(($el) => {
+      console.log('element', $el[0])
+      const rect = $el[0].getBoundingClientRect() // https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect
+      const isInViewport = (
+        rect.top >= -2 &&
+        rect.left >= 0 &&
+        rect.bottom <= Cypress.$(cy.state('window')).height() &&
+        rect.right <= Cypress.$(cy.state('window')).width()
+      )
+      expect(isInViewport).to.be.true
+    })
+  })
```

Comment:
We need to test on a mutation that has a lot of stuff in the form so that overflows. Unfortunately with the mocked mutations all of them are very short so they don't overflow, and so this test passes on master.

Also, there's no need to manually check the element is within the viewport; the assertion `.should('be.visible')` does this for you.

Here's how I would go about this
```suggestion
    cy.get('.c-interactive:first')
      .click()
      .get('#less-more-button')
      .click()
      .get('.c-mutation-menu-list')
      // choose mutation with a long form so it overflows
      .contains('Edit Runtime')
      .click()
      .get('.v-card-actions')
      .should('be.visible')
```